pcsx2/Pcsx2Config.cpp

```diff
@@ -893,7 +893,7 @@ void Pcsx2Config::GSOptions::LoadSave(SettingsWrapper& wrap)
 	// ~51x would the upper bound here for 32768x32768 textures, but you'll run out VRAM long before then.
 	UpscaleMultiplier = std::clamp(UpscaleMultiplier, 0.5f, 50.0f);
 
-	SettingsWrapIntEnumEx(HWMipmap, "mipmap_hw");
+	SettingsWrapBitBoolEx(HWMipmap, "mipmap_hw");
```

Comment:
This is going to be... problematic for existing configs, where it's not necessarily 0/1 (probably -1).

I'd suggest changing the field name to `HWMipmap`, to give everyone a new default.